src/module/data/actor-character.mjs

```diff
@@ -85,7 +85,26 @@ export default class GrimwildCharacter extends GrimwildActorBase {
 		const rollData = this.getRollData();
 
 		if (options?.stat && rollData?.stats?.[options.stat]) {
-			const formula = `{(@stats.${options.stat})d6kh, (@thorns)d8}`;
+
+			const content = await renderTemplate('systems/grimwild/templates/dialog/stat-roll.hbs', {
+				diceDefault: rollData?.stats?.[options.stat],
+				thornsDefault: rollData?.thorns
+			});
+			const rollDialog = await foundry.applications.api.DialogV2.wait({
+				window: { title: "Grimwild Roll" },
+				content,
+				modal: true,
+				buttons: [
+				{
+				  label: "Roll",
```

Comment:
Let's translate this with `game.i18n.localize()` or `game.i18n.format()`. You can find the list of translations provided by core by outputting `game.i18n.translations` in your console, and if you can't find one for this, let's add one to our en.yml file.